### PR TITLE
Non-substantive refinement of Presentations section.

### PR DESCRIPTION
**Pull request by msporny**
Related to #488.

I have tried to be very careful to not make substantive changes, but rather rephrase this section to make the normative statements in the same way that other sections of the specification make the same sort of normative statements. Please double-check my changes to make sure I'm not stating anything new (or changing the existing normative statements in a way that is substantive or would change implementations.).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/547.html" title="Last updated on Apr 24, 2019, 3:54 AM UTC (8fef5c3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/547/2dde7d4...8fef5c3.html" title="Last updated on Apr 24, 2019, 3:54 AM UTC (8fef5c3)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2250,22 +2273,22 @@ <h3>Refreshing</h3>
...
 the <a>holder</a> and the <a>verifier</a> to perform future updates of the credential.
         </p>
         <p>
-The refresh service is only expected to be used when either the credential 
-has expired, or when the issuer does not publish credential status information. 
-It is ill advised for issuers to put the <code>refreshService</code> in a 
-<a>verifiable credential</a> that does not contain public information, 
-or whose refresh service is not protected in some way. 
+The refresh service is only expected to be used when either the credential
+has expired, or when the issuer does not publish credential status information.
```
Comment (dlongley) [inline index.html]:
```suggestion
has expired or when the issuer does not publish credential status information.
```

File: index.html [hunk 2]
```diff
@@ -1642,26 +1642,49 @@ <h3>Status</h3>
         <h3>Presentations</h3>
 
         <p>
-<a>Verifiable presentations</a> MAY be used to combine and present
-<a>verifiable credentials</a>. They are packaged in such a way that the
-authorship of the data is <a>verifiable</a>.
+<a>Presentations</a> MAY be used to combine and present <a>credentials</a>.
+They are often packaged in such a way that the authorship of the data is
+<a>verifiable</a>. The data in a <a>presentation</a> is often about the
+same <a>subject</a>, but was issued by multiple <a>issuers</a>.
```
[the remark this answers is not in the document]
Comment (brentzundel) [reply index.html]: I think the following sentence handles the "from multiple credentials/issuers" case. I think this sentence might be simpler if it were simply "The data in a presentation is often about the same subject, but there is no limit to the number of subjects or issuers in the data. The aggregation of ..."

File: index.html [hunk 3]
```diff
@@ -1642,26 +1642,49 @@ <h3>Status</h3>
...
-The data in a <a>presentation</a> is often about the same <a>subject</a>, but
-was issued by multiple <a>issuers</a>. The aggregation of information from
-multiple <a>verifiable credentials</a> is a typical use of
-<a>verifiable presentations</a>.
-        </p>
+        <dl>
+          <dt><var>id</var></dt>
+          <dd>
+The <code>id</code> <a>property</a> is optional and MAY be used to provide a
+unique identifier for the presentation.
```
[the remark this answers is not in the document]
Comment (brentzundel) [reply index.html]: It is covered in the identifiers section, but I liked that it was re-emphasized here.

[the remark this answers is not in the document]
Comment (msporny) [reply index.html]: In general, we shouldn't re-state normative text in different sections of the spec, which is why I refer to the "identifier" section in the PR.

File: index.html (hunk 1 above)
Comment (TallTed) [reply index.html]: for clarity and grammar...

"when either the credential has expired or when the issuer" 

-> "when either (1) the credential has expired or (2) the issuer"
or
-> "either when the credential has expired or when the issuer" 

I prefer the former.

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (TallTed) [reply index.html]: "often about the same subject" -> "often all about one subject"

File: index.html [hunk 4]
```diff
@@ -1642,26 +1642,49 @@ <h3>Status</h3>
         <h3>Presentations</h3>
 
         <p>
-<a>Verifiable presentations</a> MAY be used to combine and present
-<a>verifiable credentials</a>. They are packaged in such a way that the
-authorship of the data is <a>verifiable</a>.
+<a>Presentations</a> MAY be used to combine and present <a>credentials</a>.
+They are often packaged in such a way that the authorship of the data is
```
[the remark this answers is not in the document]
Comment (msporny) [reply index.html]: Fixed in https://github.com/w3c/vc-data-model/pull/547/commits/34c6c5c061cc1a418e3579fce46003d38f392655.

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (msporny) [reply index.html]: Fixed in https://github.com/w3c/vc-data-model/pull/547/commits/34c6c5c061cc1a418e3579fce46003d38f392655 using @brentzundel and @TallTed's suggestions.

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html]: Fixed in https://github.com/w3c/vc-data-model/pull/547/commits/34c6c5c061cc1a418e3579fce46003d38f392655.

**Comment by msporny** [comment]
From https://www.w3.org/2019/04/09-vcwg-minutes.html#resolution06: 

RESOLUTION: Implementers were polled before the specification entered the Candidate Recommendation phase and more than two implementers stated that they would implement this feature, thus it is not at risk. Interoperability for this feature is achieved if at least two independent implementations support the data model features in the specification, The WG acknowledges that there are several different types of zero-knowledge proof mechanisms and has provided a mechanism where one or more of those mechanisms can be combined with the data model in the specification. The group agrees that non-substantive changes should be made to the section to clarify the normative requirements more precisely and then issue #488 should be closed after that PR is approved and merged.

**Comment by msporny** [comment]
Non-substantive, multiple independent reviews, all suggestions have been incorporated, WG resolution, merging.

**Pull request merged**